src/Exception/RequestException.php

```diff
@@ -11,15 +13,11 @@
  */
 class RequestException extends TransferException implements RequestExceptionInterface
 {
-    /** @var RequestInterface */
-    private $request;
+    use RequestAwareTrait, HandlerContextAwareTrait;
```

Comment:
If we are following PSR-12 and/or Symfony code style guidelines, then this needs to be:

```php
use RequestAwareTrait;
use HandlerContextAwareTrait;
```

Reply:
Fixed in 8ad0f9d